hazelcast/src/main/java/com/hazelcast/jet/core/processor/SourceProcessors.java

```diff
@@ -142,28 +142,6 @@ public static <T, K, V> ProcessorMetaSupplier streamMapP(
         return StreamEventJournalP.streamMapSupplier(mapName, predicateFn, projectionFn, initialPos, eventTimePolicy);
     }
 
-    /**
```

Comment:
I would consider those a public API and it's not `@Beta`, so we shouldn't remove it

Reply:
The class is  public but I am not sure if it can be considered public API. Because it can only be used by somebody who knows how to add processor to DAG manually. 

Let's hope that nobody is using these methods. Because currently we have switched the API to parameter object.

Reply:
The core API is not recommended one to use ootb, but still an API. Please add a breaking change notice to the PR, IMHO that should be enough indeed.